RELEASE.md

```diff
@@ -42,244 +42,255 @@ Twitter the Crosslinking the announcement to the following Discord channels.
 
 1. Checkout the correct branch
 
-a. For the first release of a new `LTS`, create a new branch from `origin/release`
```

Comment:
Are the whitespace changes on purpose?

Reply:
Ya provides a bit more clear indentation.  Lmk what you think though!

PR - https://github.com/emberjs/data/blob/7cd13169f9697db85638951813914c76379bb5cb/RELEASE.md

master - https://github.com/emberjs/data/blob/master/RELEASE.md